cc.newspring.Apollos/Workflow/Action/APISync.cs

```diff
@@ -23,20 +23,33 @@ public class APISync : Rock.Workflow.ActionComponent
         public override bool Execute( Rock.Data.RockContext rockContext, WorkflowAction action, object entity, out List<string> errorMessages )
         {
             errorMessages = new List<string>();
+            var castedModel = entity as IModel;
 
-            var model = (IModel)entity;
-            var restUserGuid = GetAttributeValue( action, "RestUser" ).AsType<Guid?>();
-
-            if ( model.ModifiedByPersonAliasId.HasValue )
+            if ( castedModel != null )
             {
-                var modifier = new PersonAliasService( new RockContext() ).Get( model.ModifiedByPersonAliasId.Value );
+                var restUserGuid = GetAttributeValue( action, "RestUser" ).AsType<Guid?>();
 
-                if ( modifier.Guid == restUserGuid )
+                if ( castedModel.ModifiedByPersonAliasId.HasValue )
                 {
-                    return true;
+                    var modifier = new PersonAliasService( new RockContext() ).Get( castedModel.ModifiedByPersonAliasId.Value );
+
+                    if ( modifier.Guid == restUserGuid )
```

Comment:
Can you leave a comment here so it's clear we're short-circuiting the sync when the change is from Apollos?  Took me over a minute to figure out what this was doing.